### ISSUE TITLE
no raw text rule fail

### ISSUE THREAD

**Issue by tomdyqin** [Intellicode/eslint-plugin-react-native#266]
{
      code: `
        export default class MyComponent extends Component {
          render() {
            const styles = StyleSheet.create({})
            return (<View>
              <Text>text1</Text>
              <Text style={styles.a}>text2</Text>
            </View>);
          }
        }
      `,
    },

this is my test code, line `text1` is ok， but `text2` use style props is fail
![1](https://user-images.githubusercontent.com/11284676/90855452-3ae52100-e3b2-11ea-94f0-ce71887689d8.png)

**Comment by alexisbronchart** [comment]
> Did this happen after the latest release?

Same issue here. Indeed after upgrading to 3.9.0

**Comment by adrianocola** [comment]
This seems to be related with #247 . 

This PR was created to allow components that use dot notation to work (ie: `Animated.Text`). This rule works by checking if the component name is in a list of allowed components that can contain text. But the function used to get the component name now is also appending the prop names to the component name.

This happens in this function:
https://github.com/Intellicode/eslint-plugin-react-native/blob/886999a1d3b068a32fc273b08dbef3fcad713a08/lib/rules/no-raw-text.js#L10-L24

Changing this section fixes the problem:
```javascript
      traverse(element, {
        JSXIdentifier({ node: identifier }) {
          if (identifier.parent.type === 'JSXOpeningElement' || identifier.parent.type === 'JSXMemberExpression') {
            identifiers.push(identifier.name);
          }
        },
      }, scope);
```

I'll make some more tests and send a new PR to fix this issue.
